docs/_docs/testnet/starting-a-node.md

```diff
@@ -0,0 +1,51 @@
+---
+---
+# Starting a node
+How to start a node in the Evan's Cat testnet.
+
+## Requirements
+- SGX enabled VM
+- Docker
+
+## Steps
+#### - Create an SGX enabled VM
+Recommended Standard DC4s v2 (4 vcpus, 16 GiB memory) in Azure.
+
+#### - Install Docker
+
+```
+sudo apt-get update \
+    && sudo apt-get install -y jq \
+    && curl -fsSL https://get.docker.com -o get-docker.sh && sh ./get-docker.sh
+```
+
+#### - Download Obscuro Start script
+
+Make sure to use the latest version at https://github.com/obscuronet/go-obscuro/tags
+
+```
+  wget https://raw.githubusercontent.com/obscuronet/go-obscuro/v0.7.1/testnet/start-obscuro-node.sh && \
+  wget https://raw.githubusercontent.com/obscuronet/go-obscuro/v0.7.1/testnet/docker-compose.yml
+```
+
+#### - Start Obscuro Node
+
+To start the obscuro node some information is required to populate the starting script.
+
+- (p2p_public_address) Local address for the Peer to Peer connections. Must be open to outside connections.
+- (pkstring) Private Key to issue transactions into the Layer 1
+- (host_id) Public Key derived from the Private Key
```

Comment:
Given the host_id is derived from the private key, why does the user have to provide as it can be done programatically?

Reply:
Because the enclave requires it to ensure the host is properly connected and attested. And starting the enclave is a separate process from starting the host.

Maybe the enclave can receive the L1 private key and deduce the same HostID ? I didn't give it much importance tbh.

Reply:
Fair enough